packages/manager/apps/pci-rancher/src/components/layout-helpers/Dashboard/RancherDetail/RancherDetail.component.tsx

```diff
@@ -42,6 +51,9 @@ import { useTrackingAction } from '@/hooks/useTrackingPage/useTrackingPage';
 import { getLatestVersionAvailable } from '@/utils/rancher';
 import { TrackingEvent, TrackingPageView } from '@/utils/tracking';
 
+// to delete when return from API is OK
+const egressCIDRBlocksFake = ['192.0.2.0/24', '198.51.100.0/24'];
```

Comment:
As the list is hard coded, waiting for the API, I'll suggest to rename it `egressCIDRBlocks` because production code should not give away that the data source is fake.